code/game/gamemodes/vampire/vampire_powers.dm

```diff
@@ -219,8 +219,10 @@
 /obj/effect/proc_holder/spell/vampire/mob_aoe/glare/cast(list/targets, mob/user = usr)
 	user.visible_message("<span class='warning'>[user]'s eyes emit a blinding flash!</span>")
 	if(istype(user:glasses, /obj/item/clothing/glasses/sunglasses/blindfold))
-		to_chat(user, "<span class='warning'>You're blindfolded!</span>")
-		return
+		var/obj/item/clothing/glasses/sunglasses/blindfold/B = user:glasses
+		if(B.tint)
+			to_chat(user, "<span class='warning'>You're blindfolded!</span>")
+			return
 	for(var/mob/living/target in targets)
```

Comment:
Would you mind changing this bit of code to avoid the use of `:`? It was like this already so it's not an obligation, but it'd be good to update the code to be in line with the [Contribution Guidelines](https://github.com/ParadiseSS13/Paradise/blob/master/.github/CONTRIBUTING.md#No-overriding-type-safety-checks).
I can't leave a github suggestion on this bit so you'll have to type it in manually:
```dm
/obj/effect/proc_holder/spell/vampire/mob_aoe/glare/cast(list/targets, mob/user = usr)
	user.visible_message("<span class='warning'>[user]'s eyes emit a blinding flash!</span>")
	if(ishuman(user))
		var/mob/living/carbon/human/H = user
		if(istype(H.glasses, /obj/item/clothing/glasses/sunglasses/blindfold))
			var/obj/item/clothing/glasses/sunglasses/blindfold/B = H.glasses
			if(B.tint)
				to_chat(user, "<span class='warning'>You're blindfolded!</span>")
				return

	for(var/mob/living/target in targets)
```